mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -384,12 +388,12 @@ def _id_counter(self):
             yield i
             i += 1
 
-    def set_verbose_logging(self, verbose: bool = False):
+    def set_snippet_client_verbose_logging(self, verbose=False):
+        # type: (bool) -> None
         """Switches verbose logging. True for logging full RPC response.
 
```

Comment:
need to explain what "verbose" means in this docstring.
Especially need to highlight the difference being whether the `value` field of the response message from snippet server is fully logged.

Reply:
Add more description in docstring.